index.js

```diff
@@ -142,7 +193,13 @@ function getAllMoviesReleasedAtOrBeforeYear() {}
  *  getBiggestBoxOfficeMovie(movies);
  *  //> "Incredibles 2"
  */
-function getBiggestBoxOfficeMovie() {}
+function getBiggestBoxOfficeMovie(movies) {
+  let newArr = []
+  // for (let i = 0; i < movies.length; i ++){
+  //   if (movies[i].boxOffice > newArr){
+return null
+  //   }
+}
```

Comment:
Take another stab at this!  We know that our input is a `string`  that has some extra characters in it `$ and ,`.  How can we manipulate this string and then compare the values using the accumulator pattern to find the highest result?